<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pxtone collab</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/favicon/icon</normaloff>:/favicon/icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewHost"/>
    <addaction name="actionOpenHost"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionRender"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionShortcuts"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionIncrease_font_size"/>
    <addaction name="actionDecrease_font_size"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionCopyOptions"/>
    <addaction name="actionClean"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="actionClear_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenHost">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save locally</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionNewHost">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionClearSettings">
   <property name="text">
    <string>Clear app settings</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionIncrease_font_size">
   <property name="text">
    <string>Increase font size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionDecrease_font_size">
   <property name="text">
    <string>Decrease font size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionDark_piano_roll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark piano roll</string>
   </property>
  </action>
  <action name="actionAuto_add_unit">
   <property name="text">
    <string>Auto-add unit</string>
   </property>
  </action>
  <action name="actionChordPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview with all units</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>Render</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClear_Settings">
   <property name="text">
    <string>Clear Settings</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean units / voices</string>
   </property>
  </action>
  <action name="actionVelocity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Velocity</string>
   </property>
  </action>
  <action name="actionVoice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Voice</string>
   </property>
  </action>
  <action name="actionCopyOptions">
   <property name="text">
    <string>Copy options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
